game/card.go

```diff
@@ -73,6 +77,23 @@ func newCardHelper(name CardName) *Card {
 			CastingCost:   &CastingCost{Colorless: 2},
 			IsCreature:    true,
 		}
+	case NestInvader:
+		tokenCard := &Card{
```

Comment:
I think if you nest it like this, you wont be able to call NewCard(EldraziSpawnToken). Can you just have EldraziSpawnToken be its own card at the top level of this switch?